xml/admin_logging.xml

```diff
@@ -0,0 +1,446 @@
+      Updating settings
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Upgrading a cluster
+     </para>
+    </listitem>
+   </itemizedlist>
+
+<screen>&prompt.user;<command>docker exec -it $(docker ps | grep dashboard | awk '{print $1}') \
+entrypoint.sh bundle \
+exec rails runner 'puts(Minion.all.map(&amp;:to_yaml))' &gt; events.txt</command>
+
+<command>/var/lib/supportutils-plugin-suse-caasp/debug-salt \
+--json_output=events.txt \
+--summary_output=events-summarized.txt \
+--text-status \
+--no-color</command>
+         </screen>
+
+   <para>
+    Reading the <filename>events-summarized.txt</filename> file
+    should be enough for detecting most (if not all) of the issues caused by
+    <literal>Salt</literal>.
+   </para>
+  </sect2>
+
+  <sect2 xml:id="sec.admin.logging.salt.master">
+   <title>Salt Master Log</title>
+   <para>
+    Retrieve the <literal>Salt</literal> master logs. This doesn't usually
+    provide much insight, but might contain some issues related to the
+    <literal>salt-database</literal> connection.
+   </para>
+
+<screen>&prompt.user;<command>docker exec -it $(docker ps | grep salt-master | awk '{print $1}') \
+cat /var/log/salt/master</command>
+    </screen>
+
+  </sect2>
+
+  <sect2 xml:id="sec.admin.logging.salt.minion">
+   <title>Salt Minion Logs</title>
+   <para>
+    Retrieve the <literal>salt-minion</literal> logs for all nodes. This will
+    show all output for all <literal>salt-minions</literal> at once. Execute the
+    following command on the admin node.
+   </para>
+   <para>
+    Of course, it's possible to retrieve this information on any specific node
+    by reading the <filename>/var/log/salt/minion</filename> file.
+   </para>
+<screen>
+&prompt.user;<command>docker exec -it $(docker ps | grep salt-master | awk '{print $1}') \
+salt '*' cmd.run "cat /var/log/salt/minion"</command>
+    </screen>
+
+  </sect2>
+ </sect1>
+
+ <sect1 xml:id="sec.admin.logging.transactional-updates">
+  <title>Transactional Update Log</title>
+  <para>
+   In some cases, <command>transactional-update</command> can run into issues
+   when creating new snapshots. These problems can stem from problems attempting
+   to download packages from the update channels or when nodes run out of disk
+   space.
+  </para>
+
+<screen>
+&prompt.user;<command>docker exec -it $(docker ps | grep salt-master | awk '{print $1}') \
+salt -P 'roles:(admin|kube-master|kube-minion)' \
+cmd.run "journalctl -u transactional-update"</command>
+ </screen>
+ </sect1>
+
+ <sect1 xml:id="sec.admin.logging.fluentd">
+  <title>External log collection (fluentd)</title>
+  <para>
+   There's a notable exception at infra level, and is that we allow audit logs
+   to be configured from within Velum. This allows the audit logs to be written
+   on the &kube; master nodes at <filename>/var/log/kube-apiserver/audit.log</filename>
+   and an external data collector (continuous reading) like <literal>fluentd</literal>
```

Comment:
That one came from you :) I just rewrote the word to continuous because that made more sense. Apparently `continuously reading` is the correct thing to write here.

Reply:
Ah, yeah, that was a `(continue reading)` to continue reading the next paragraph, wasn't intended to be included in the documentation. Can be removed then.